core/file/dicom/image.cpp

```diff
@@ -107,6 +107,12 @@ namespace MR {
               case 0x1314U:
                 flip_angle = item.get_float (0, flip_angle);
                 return;
+              case 0x9034U:
+                if (item.get_string (0) == "LINEAR")
+                  pe_sign = -1;
+                else if (item.get_string (0) == "REVERSE_LINEAR")
+                  pe_sign = 1;
+                return;
```

Comment:
This is writing to an existing class member?

If so, Siemens data may provide such information *both* in the CSA and here, and there is some chance that they may conflict.

In other contexts where this occurs (slice timing comes to mind), I store the data that are obtained from all possible sources, and later code is then responsible for determining which of the possible sources are available, which takes priority, potentially when to issue a warning if there is inconsistency.

Reply:
Resolving as modification of handling of phase encoding information is being redirected to a different PR.